fontforge/splineutil2.c

```diff
@@ -163,28 +163,19 @@ return( v2-v1 > re );
 }
 
 int RealNear(real a,real b) {
```

Comment:
While we're here, why not change the return type to `bool`? I've been trying to do that where possible.